setup.py

```diff
@@ -36,5 +36,5 @@
         'Topic :: Internet :: WWW/HTTP',
     ],
     test_suite='tests',
-    tests_require=['mock']
+    tests_require=['mock', 'pytest']
```

Comment:
Aren't `python setup.py test` and `tox` two alternative entry points to running our tests, and we should pick one of them?

Related: https://github.com/tox-dev/tox/issues/13